src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -158,13 +160,39 @@ public static InterpolatedStringExpressionSyntax InterpolatedString(FormattableS
 
     public static InvocationExpressionSyntax NameOf(ExpressionSyntax expression) => Invocation(_nameofIdentifier, expression);
 
+    public static ExpressionSyntax ConstantNameOf(ExpressionSyntax expression)
```

Comment:
I don't think nameof of an element access expression ever makes sense, therefore I don't think this method should be needed at all. In what case does this make sense?

Reply:
mb accidentally added it when I made a backup. I'll rmeove it